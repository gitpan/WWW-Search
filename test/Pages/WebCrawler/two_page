http://www.webcrawler.com/cgi-bin/WebQuery?perPage=84&start=0&search=Lando&showSummary=true
http://www.webcrawler.com/cgi-bin/WebQuery?perPage=84&start=84&search=Lando&showSummary=true
http://www.webcrawler.com/cgi-bin/WebQuery?perPage=84&start=168&search=Lando&showSummary=true
