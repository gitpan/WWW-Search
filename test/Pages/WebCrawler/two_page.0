<HTML>
<HEAD>
<TITLE>
WebCrawler Search Results for: Lando
</TITLE>
</HEAD>
<BODY BGCOLOR="#ffffff">
<CENTER>
<MAP NAME="gnav">
<AREA SHAPE="rect" COORDS="2,1,239,43" HREF=/relocate/co=internal_navlink&|pk=1;http://www.webcrawler.com/>
<AREA SHAPE="rect" COORDS="461,28,490,40" HREF="/relocate/co=internal_navlink&|pk=1;http://www.webcrawler.com/">
<AREA SHAPE="rect" COORDS="494,29,538,41" HREF="/relocate/co=internal_navlink&|pk=1;http://my.excite.com/webcrawler/">
<AREA SHAPE="rect" COORDS="542,29,569,40" HREF="/relocate/co=internal_navlink&|pk=1;http://mailexcite.com">
<AREA SHAPE="rect" COORDS="573,28,596,40" HREF="/relocate/co=internal_navlink&|pk=1;http://www.webcrawler.com/Help/Help.html">
</MAP>
<FORM ACTION=http://www.webcrawler.com/cgi-bin/redirform>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH=600>
<TR><TD colspan=2>
<IMG SRC=/icons/bumper_results.gif ALT="WebCrawler Search Results" WIDTH=600 HEIGHT=42 BORDER=0 USEMAP="#gnav"></TD></TR>
<TR>
<td BGCOLOR=#FFCC66>&nbsp;<font face=geneva,arial size=-1><B></B></font></td>
<TD ALIGN=RIGHT BGCOLOR=#FFCC66>
<SELECT NAME="url" SIZE="1" onChange="submit()">
<OPTION VALUE="/">-- Choose a Channel --</OPTION>
<OPTION VALUE="/Arts/">Arts &amp; Books</OPTION>
<OPTION VALUE="/Auto/">Auto</OPTION>
<OPTION VALUE="http://quicken.webcrawler.com/">Business &amp; Investing</OPTION>
<OPTION VALUE="/Careers/">Careers &amp; Education</OPTION>
<OPTION VALUE="/Computers/">Computers &amp; Internet</OPTION>
<OPTION VALUE="/Entertainment/">Entertainment</OPTION>
<OPTION VALUE="/Games/">Games</OPTION>
<OPTION VALUE="/Health/">Health &amp; Fitness</OPTION>
<OPTION VALUE="/Family/">Home &amp; Family</OPTION>
<OPTION VALUE="http://my.excite.com/webcrawler/">My Page</OPTION>
<OPTION VALUE="http://nt.excite.com/webcrawler/">News</OPTION>
<OPTION VALUE="/Chat/">People &amp; Chat</OPTION>
<OPTION VALUE="/Reference/">Reference</OPTION>
<OPTION VALUE="/Relationships/">Relationships</OPTION>
<OPTION VALUE="/Shopping/">Shopping</OPTION>
<OPTION VALUE="/Sports/">Sports &amp; Recreation</OPTION>
<OPTION VALUE="http://webcrawler.city.net/">Travel</OPTION>
</SELECT>
<INPUT TYPE="hidden" NAME="src" VALUE="jumpbox-search">
<INPUT TYPE="hidden" NAME="|pk" VALUE="1">
<INPUT TYPE="submit" VALUE="GO!">
</TD></TR>
</TABLE>
</FORM>
<CENTER>
<FORM ACTION="/cgi-bin/WebQuery" METHOD="GET" NAME="wcsearchform"><NOBR>
<INPUT NAME="search" VALUE='Lando' SIZE=35>
<INPUT TYPE="submit" VALUE="Search"></NOBR>
<INPUT NAME="src" VALUE="wc_results" TYPE="hidden">
<INPUT NAME="showSummary" VALUE="true" TYPE="hidden">
</FORM>
</CENTER>
<CENTER>
<FORM
action="/relocate/ad=97266_htmlad.gif|pk=2906148;http://www.cdnow.com/from=bax:s:wcm:rgpg:a0087/pagename=/RP/CDN/FIND/displaypop.html/from=bax:s:wcm:rgpg:a0087"
method="GET">
<TABLE BORDER=1 CELLPADDING=0 CELLSPACING=0 WIDTH=468 HEIGHT=60><TR><TD>
<table width="100%" border="0"
cellspacing="0" cellpadding="0" height="60"> <tr valign="bottom"
bgcolor="#FF3300"> <td colspan="5"><img
src="http://image.eimg.com/ads/97266_find_your_music.gif" border=0 width="468" height="36"></td> </tr> <tr
valign="top"> <td bgcolor="#3366CC"
width="22" valign="top"><img src="http://image.eimg.com/ads/97266_search.gif" border=0 width="65"
height="22"></td> <td bgcolor="#3366CC"
width="127" valign="middle" align="right"> <SELECT NAME="index"> <OPTION
VALUE="a" selected>Artist <OPTION
VALUE="t">Title <OPTION VALUE="s">Song Title <OPTION VALUE="l">Record
Label <OPTION VALUE="m">Movie Title <OPTION
VALUE="r">Actor/Actress </SELECT> </td> <td bgcolor="#3366CC">&nbsp;</td>
<td bgcolor="#3366CC" width="162"
valign="top"> <INPUT TYPE="text" NAME="string" VALUE="" SIZE=20
MAXLENGTH=45> </td> <td bgcolor="#3366CC"
valign="top" align="right"><INPUT TYPE="image" src="http://image.eimg.com/ads/97266_go.gif" width="111"
height="22" border="0"></td> </tr>
</table></TD></TR></TABLE><A
HREF="/relocate/ad=97266_htmlad.gif|pk=2906148;http://www.excite.com"></A><CENTER><FONT SIZE=-1 COLOR=#0000FF
FACE=arial,
geneva><B>Find Your Music in Your Store!</B></FONT></CENTER> </form>
</CENTER>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH=100%>
<TR><TD VALIGN=top WIDTH=75%>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 width=98%>
<TR><TD>&nbsp;<FONT FACE=Arial,helvetica><B>Simple Search Results</B></FONT></TD></TR>
<TR><TD BGCOLOR=#FFCC66 HEIGHT=3><IMG SRC=/icons/Spacer.gif WIDTH=1 HEIGHT=1></TD></TR>
</TABLE>
<FONT SIZE=-1 FACE=geneva,helvetica>
<P><FONT SIZE=3><B>Web Results for:</B></FONT>&nbsp;&nbsp;
<B>Lando</B>
<BR>
(25 of 170)
- show
<A HREF="/cgi-bin/WebQuery?search=Lando&showSummary=false&start=0&perPage=25">titles</a>
for these results.
<dl>
<DT><FONT FACE="Times" COLOR="#006699"><B>62% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www-personal.umich.edu/~kathyb/joe/lando.htm>Leaving Lando Land!</A>
<DD>22 September 1997 Dear Lando Lovers, As of today the Joe Lando Web Site will cease to exist. Based on its use over the last few months, I realized that very few people were actually reading it. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1386619&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www-personal.umich.edu/~kathyb/joe/lando.htm</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>61% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://members.tripod.com/~RicoNLando/index.html>Pimps and Playas</A>
<DD>Pimpin ain't easy but...It sure is alot of fun. It's just something we got to do. My name is Steve-o, My Name is Lando. We pimp it on the southside. just taking care of buisness. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=3467&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://members.tripod.com/~RicoNLando/index.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>59% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.comnet.ca/~carolm/pixs.html>Welcome to the Joe Lando Web Page - Pix</A>
<DD>Pix "Revolutions" - Some candid pics taken in March, 1999 on the set of the DQ movie, Revolutions. "Joe Lando - Video Clips" - If you missed Joe in various performances, now's your chance to download video clips of some of his work. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1945075&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.comnet.ca/~carolm/pixs.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>59% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://pages.prodigy.com/Kewltoy/books.htm>Timeline Page, Chapter One</A>
<DD>This page will show you everything that has happened up to Star Wars IV. This Timeline has photos, sounds and movies of the characters, and places listed herein. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1942566&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://pages.prodigy.com/Kewltoy/books.htm</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>56% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.rit.edu/~dxp4044/>Welcome to the Corning Area Ski Club - Main Page</A>
<DD>The Corning Area Ski Club is a nonprofit, volunteer organization. We are a group of skiers, at all levels of ability, who live in the greater Corning/Elmira area and enjoy skiing and other outdoor activities. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1439223&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.rit.edu/~dxp4044/</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>55% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://lldesign.com/>lermechin lando design</A>
<DD>lermechin lando design is an agency of global french design. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1403155&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://lldesign.com/</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>55% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.comnet.ca/~carolm/index.html>The Joe Lando Web Page - Home</A>
<DD>Site Updated May 14, 1999 Chat online with Joe in Yahoo Monday, May 17, 1999 at 6:00 p.m. EDT *What's New* and *News Updates* updated May 6, 1999 Articles & Interviews Joe's Movies Scripts Pix Biography Messages from Joe Tribute to Sully Sights & Sounds of Joe Lando Birthday Countdown Joe's Horoscope Thanks & Links <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1346617&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.comnet.ca/~carolm/index.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>54% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.landolighting.com/>Welcome to LANDO LIGHTING GALLERIES</A>
<DD>Copyright 1998 LANDO LIGHTING GALLERIES. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=668907&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.landolighting.com/</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>53% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.angelfire.com/mo/CaptainLando/index.html>Capt. Lando's</A>
<DD>My Favorite Links Capt. Decipher Star Wars Homepage CLICK HERE TO GO TO MY FOR SALE LIST!!! Email: death_star_2@hotmail.com <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1692484&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.angelfire.com/mo/CaptainLando/index.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>53% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.geocities.com/Area51/Hollow/5249/forsale.html>Star Wars Page.</A>
<DD>I WANNA TRADE! I am always looking to trade for Star Wars items this is a list of stuff that I am currently looking to TRADE for. If you are willing to do any trading take a look at the stuff that I have, prices are different for TRADING than they are FOR SALE. e-mail me with any trade possibilities. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1937117&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.geocities.com/Area51/Hollow/5249/forsale.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>53% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www2.netdoor.com/~edmonson/actionfigurelist.html>John Edmonson's Star Wars Action Figures Page</A>
<DD>Vintage Star Wars Action Figures. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1285361&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www2.netdoor.com/~edmonson/actionfigurelist.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>52% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.yakface.com/vintage/figures.html>Figures List</A>
<DD>1 Complete Set of Star Wars Action Figures. 100, Loose &amp; complete, includes Yakface and Blue Snaggletooth - $1950.00 P=Punched U=Unpunched Kenner POTF Tatooine Skiff in SEALED never opened C9.5 box $690.00 <NOBR><A HREF="/cgi-bin/WebQuery?qbe=103691&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.yakface.com/vintage/figures.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>52% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.janeseymour.org/z/tv0599.htm>Jane Seymour Web Page - http://www.janeseymour.org - May TV ...</A>
<DD>Jane Seymour Television Schedules <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1838785&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.janeseymour.org/z/tv0599.htm</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>52% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://w3.one.net/~coleman/landoballs.html>landoballs</A>
<DD>I heard the news after I saw Return of the Jedi...Could it be true? My hero, eating balls...Well it is. Lando loves a tasty set of balls now and then. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=987018&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://w3.one.net/~coleman/landoballs.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>51% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://members.it.tripod.de/emacar/index.html>Lando's Home Page</A>
<DD>emancarr@tin.it via cordero 1 S. Polo di Torrile, PR 43056 Italy Software Disponibile Gecosys 16 : Software per la Gestione dei costi aziendali Gepo : Software per la Gestione della piccola azienda <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1215345&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://members.it.tripod.de/emacar/index.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://members.aol.com/kidcodeine/page/index.htm>A Tear in the Atmosphere</A>
<DD>A Tear in the Atmosphere My name's Shawn. I'm 24, from CT, &amp; my career goal is to become a successful filmmaker. I write, direct, and act as much as possible, and cannot wait until the day when I can do it professionally. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1696363&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://members.aol.com/kidcodeine/page/index.htm</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.landofamily.com/index.htm>Welcome to the Lando Family Network</A>
<DD>A former 60 Minutes Producer, Barry Lando with his wife,Elisabeth, and six year old son, Edward, are taking a year to travel across the Pacific, by plane, not boat. Follow the trip and also join their quest to find a <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1111478&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.landofamily.com/index.htm</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.duke.edu/~tpl/bollweevils.html>The Bollweevils Page!</A>
<DD>Welcome to The Bollweevils homepage! In case you don't already know, the Bollweevils were one of the greatest bands to ever emerge from the streets of Chicago. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=872305&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.duke.edu/~tpl/bollweevils.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.maxpages.com/dons>Entertainment -- Dons Star Wars Webpage -- http://www.maxpag...</A>
<DD>starwars, the force, anakin, Episode I, jedi sith,obi-wan, <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1007555&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.maxpages.com/dons</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.maxpages.com/dons/don_miles>Entertainment -- ALL ABOUT DON MILES! -- http://www.maxpages...</A>
<DD>Don, Don Miles, freaks, losers, dorks <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1840615&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.maxpages.com/dons/don_miles</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.stwing.upenn.edu/starwars/films/rotj.txt>Untitled</A>
<DD>S T A R W A R S EPISODE VI REVENGE OF THE JEDI BY LAWRENCE KASDAN AND GEORGE LUCAS FROM THE NOVEL BY GEORGE LUCAS SECOND DRAFT - DECEMBER 19, 1982 1 SPACE 1 The boundless heavens serve as a back-drop for the MAIN TITLE, followed by a ROLL-UP, which crawls into infinity. <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1205286&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.stwing.upenn.edu/starwars/films/rotj.txt</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.janeseymour.org/dq/dqmovie1.htm>Dr. Quinn, Medicine Woman: The Movie</A>
<DD>Jane Seymour, Dr. Quinn Medicine Woman: The Movie <NOBR><A HREF="/cgi-bin/WebQuery?qbe=214534&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.janeseymour.org/dq/dqmovie1.htm</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>50% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.roughcut.com/star_wars/rewind/index.html>Rewind- Star Wars Special Edition Issue</A>
<DD>Week ending February 2, 1997 Numbers in parentheses indicate weeks on chart. Click the titles in red to see what we thought. PHENOMENON (3) KINGPIN (4) <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1331809&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.roughcut.com/star_wars/rewind/index.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>49% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://members.tripod.com/~dasasc/index.html>Star Wars Items For Sale: All At Reduced Prices</A>
<DD>A wonderful collection of the Star Wars action figures, cases, playsets, ships, and more, all at reduced costs for sale by owner <NOBR><A HREF="/cgi-bin/WebQuery?qbe=747010&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://members.tripod.com/~dasasc/index.html</FONT>
<DT><FONT FACE="Times" COLOR="#006699"><B>49% </B></FONT>
&nbsp;&nbsp;
<A HREF=http://www.dislessici.org/>Welcome to dislessici - May 16, 1999
</A>
<DD>linux/bsd networking tools to secure and improve your box. an easy way to keep your os up to date. also many projects about hacking <NOBR><A HREF="/cgi-bin/WebQuery?qbe=1790663&showSummary=true&start=0&perPage=25">Similar Pages</A></NOBR>
<BR><FONT SIZE="-2">http://www.dislessici.org/</FONT>
</dl>
<BR>
<FORM ACTION="/cgi-bin/WebQuery" method="GET">
<INPUT TYPE="hidden" NAME="search" VALUE='Lando'>
<INPUT TYPE="hidden" NAME="src" VALUE="wc_more">
<INPUT TYPE="hidden" NAME="start" VALUE="25">
<INPUT TYPE="hidden" NAME="showSummary" VALUE="true">
<INPUT TYPE="hidden" NAME="perPage" VALUE=25>
<INPUT TYPE="submit" VALUE="Get the next 25 results">
</FORM>
<P><FONT SIZE=-2 FACE=geneva,helvetica>
For more comprehensive results, try <A HREF=http://www.excite.com/search.gw?search=Lando>Excite Search</A>!
</FONT>
</FONT> </TD><TD VALIGN=top ALIGN=left WIDTH=30%>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH=100%>
<TR><TD ALIGN=left>&nbsp;<FONT FACE="Arial,Helvetica"><B>Shortcuts</B></FONT></TD></TR>
<TR><TD BGCOLOR=#FFCC66 HEIGHT=3><IMG SRC=/icons/Spacer.gif WIDTH=1 HEIGHT=1></TD></TR></TABLE>
<TABLE BORDER=0 CELLPADDING=2 CELLSPACING=1 BGCOLOR=#ffffcc WIDTH=100%>
<TR><TD ALIGN=center VALIGN=top>
<!-- promo shortcuts -->
<IMG SRC="/img/web/icons/barnes5.gif"><BR>
<FONT FACE="geneva,Helvetica" SIZE="-1">
<A HREF="/relocate/co=wcs_barnes;http://barnesandnoble.bfast.com/booklink/click?sourceid=17288&is_search=Y&keyword=Lando&match=partial&options=and">Lando</A><BR>
</FONT><BR>
<!-- end shortcuts -->
</TD></TR>
</TABLE>
</TD></TR>
</TABLE>
<BR>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH=100%>
<TR><TD>&nbsp;<FONT FACE=Arial,helvetica><B>Search Again</B></FONT></TD></TR>
<TR><TD BGCOLOR=#FFCC66 HEIGHT=3><IMG SRC=/icons/Spacer.gif WIDTH=1 HEIGHT=1></TD></TR>
</TABLE>
<FORM ACTION="/cgi-bin/WebQuery" NAME=wcsearchform>
<INPUT TYPE=hidden NAME="showSummary" VALUE="true">
<INPUT NAME="search" VALUE='Lando' SIZE=35>
<INPUT TYPE="submit" VALUE="Search">
</FORM>
<BR>
<CENTER>
<FORM
action="/relocate/ad=97266_htmlad.gif|pk=2906148;http://www.cdnow.com/from=bax:s:wcm:rgpg:a0087/pagename=/RP/CDN/FIND/displaypop.html/from=bax:s:wcm:rgpg:a0087"
method="GET">
<TABLE BORDER=1 CELLPADDING=0 CELLSPACING=0 WIDTH=468 HEIGHT=60><TR><TD>
<table width="100%" border="0"
cellspacing="0" cellpadding="0" height="60"> <tr valign="bottom"
bgcolor="#FF3300"> <td colspan="5"><img
src="http://image.eimg.com/ads/97266_find_your_music.gif" border=0 width="468" height="36"></td> </tr> <tr
valign="top"> <td bgcolor="#3366CC"
width="22" valign="top"><img src="http://image.eimg.com/ads/97266_search.gif" border=0 width="65"
height="22"></td> <td bgcolor="#3366CC"
width="127" valign="middle" align="right"> <SELECT NAME="index"> <OPTION
VALUE="a" selected>Artist <OPTION
VALUE="t">Title <OPTION VALUE="s">Song Title <OPTION VALUE="l">Record
Label <OPTION VALUE="m">Movie Title <OPTION
VALUE="r">Actor/Actress </SELECT> </td> <td bgcolor="#3366CC">&nbsp;</td>
<td bgcolor="#3366CC" width="162"
valign="top"> <INPUT TYPE="text" NAME="string" VALUE="" SIZE=20
MAXLENGTH=45> </td> <td bgcolor="#3366CC"
valign="top" align="right"><INPUT TYPE="image" src="http://image.eimg.com/ads/97266_go.gif" width="111"
height="22" border="0"></td> </tr>
</table></TD></TR></TABLE><A
HREF="/relocate/ad=97266_htmlad.gif|pk=2906148;http://www.excite.com"></A><CENTER><FONT SIZE=-1 COLOR=#0000FF
FACE=arial,
geneva><B>Find Your Music in Your Store!</B></FONT></CENTER> </form>
</CENTER>
<P>
<hr width=600 size=1 noshade>
<table border=0 cellpadding=0 cellspacing=0 width=600>
<tr><td valign=top align=center>
<FONT FACE="geneva,Arial,Helvetica" SIZE="-1">
<b>Services</b> :&nbsp;&nbsp;
<A HREF=/relocate/co=7723;http://yellowpages.zip2.com/webc>yellow&nbsp;pages</A>
&nbsp;&nbsp;<A HREF=/relocate/co=map-searchlink;http://webcrawler.city.net/indexes/top_maps.dcg>maps</A>
&nbsp;&nbsp;<A HREF=/relocate/co=pfinder-searchlink;http://www.whowhere.com/WebCrawler/wc_search.html>people&nbsp;finder</A>
&nbsp;&nbsp;<A HREF=/relocate/co=shopping-searchlink;http://jango.excite.com/wc/>simple shopping</A>
&nbsp;&nbsp;<A HREF=/relocate/co=horo-searchlink;http://my.excite.com/horoscopes/webcrawler/>horoscopes</A>
<BR><A HREF=/relocate/co=c2k-searchlink;http://www.classifieds2000.com/cgi-cls/display.exe?partner=webcrawler&path=class&template=none&>classifieds</A>
&nbsp;&nbsp;<A HREF=/relocate/co=quotes-searchlink;http://quicken.webcrawler.com/investments/tickersearch/>stock&nbsp;quotes</A>
&nbsp;&nbsp;<A HREF=/relocate/co=weather-searchlink;http://my.excite.com/webcrawler/weather/>weather</A>
<P><A HREF="http://www.webcrawler.com/relocate/co=internal_footernav&|pk=1;http://www.webcrawler.com/">home</A>&nbsp;||&nbsp;
<A HREF="http://www.webcrawler.com/relocate/co=internal_footernav&|pk=1;http://my.excite.com/webcrawler/">my page</A>&nbsp;||&nbsp;
<A HREF="http://www.webcrawler.com/relocate/co=internal_footernav&|pk=1;http://mailexcite.com/">email</A>&nbsp;||&nbsp;
<A HREF="http://www.webcrawler.com/relocate/co=internal_footernav&|pk=1;http://www.webcrawler.com/Help/Help.html">help</A></FONT>
</TD></tr></table>
<P>
<FONT FACE=GENEVA,ARIAL size=-1>
<A HREF=/Help/BookmarkWC.html>Bookmark&nbsp;WebCrawler</A></font>
&nbsp;&#183;&nbsp;<font face=geneva,arial size=-1><A HREF=http://corp.excite.com/>About Excite</A></font>
&nbsp;&#183;&nbsp;<font face=geneva,arial size=-1><A HREF=http://www.webcrawler.com/info/advertising/>Advertise&nbsp;on&nbsp;WebCrawler</A></font>
&nbsp;&#183;&nbsp;<font face=geneva,arial size=-1><A HREF=/Help/GetListed/AddURLS.html>Add&nbsp;your&nbsp;URL</A>
</FONT>
<BR>
<FONT FACE="geneva,Arial,Helvetica" COLOR="#000000" SIZE="-2">&nbsp;copyright &copy; 1999 </FONT><A HREF="http://corp.excite.com/"><FONT FACE="Arial,Helvetica" COLOR="#FF0000" SIZE="-2">Excite, Inc.</FONT></A>
</CENTER>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_WebCrawler_1x1_RunOfSite_Any&event=KeywordResults&time=1999.06.30.11.40.10" HEIGHT=1 WIDTH=1 BORDER=0>
</BODY>
</HTML>
<!-- timestamp: 1 0.141529 0.141529 -->
