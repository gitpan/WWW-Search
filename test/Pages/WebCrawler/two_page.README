This set of test-result pages was created on Wed Jun 30 11:40:15 1999
