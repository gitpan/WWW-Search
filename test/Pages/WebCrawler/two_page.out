http://www-personal.umich.edu/~kathyb/joe/lando.htm
http://members.tripod.com/~RicoNLando/index.html
http://www.comnet.ca/~carolm/pixs.html
http://pages.prodigy.com/Kewltoy/books.htm
http://www.rit.edu/~dxp4044/
http://lldesign.com/
http://www.comnet.ca/~carolm/index.html
http://www.landolighting.com/
http://www.angelfire.com/mo/CaptainLando/index.html
http://www.geocities.com/Area51/Hollow/5249/forsale.html
http://www2.netdoor.com/~edmonson/actionfigurelist.html
http://www.yakface.com/vintage/figures.html
http://www.janeseymour.org/z/tv0599.htm
http://w3.one.net/~coleman/landoballs.html
http://members.it.tripod.de/emacar/index.html
http://members.aol.com/kidcodeine/page/index.htm
http://www.landofamily.com/index.htm
http://www.duke.edu/~tpl/bollweevils.html
http://www.maxpages.com/dons
http://www.maxpages.com/dons/don_miles
http://www.stwing.upenn.edu/starwars/films/rotj.txt
http://www.janeseymour.org/dq/dqmovie1.htm
http://www.roughcut.com/star_wars/rewind/index.html
http://members.tripod.com/~dasasc/index.html
http://www.dislessici.org/
http://maxpages.com/theforce/Outdoor_Bar
http://users.ids.net/~mtavares/oldchewie.html
http://www.the-cantina.com/library/swdrink.html
http://www.gis.net/~mcfadden/jmm/swchron4.html
http://www.reidenbaugh.com/
http://www.specialweb.com/tv/actors.html
http://linux.microbsys.com/
http://www.bmi.net/pfries/swdrink.html
http://starwars.jacksonville.com/
http://www.sfsite.com/depts/wars01.htm
http://www.angelfire.com/al/babcom5/b5.html
http://www.geocities.com/Area51/Realm/3175/
http://pages.whowhere.com/entertainment/joebount/have.html
http://www.twilightonline.com/
http://linuxfab.cx/
http://web.nmsu.edu/~rtelles/swdrink.html
http://www.conch-cigars.com/
http://www.cbourne.com/
http://www.sp50.com/topsites/rankem.html
http://www.suzann.com/soaps/oltl/oltl.htm
http://homepages.infoseek.com/~label98/bands.htm
http://augustachronicle.com/starwars/
http://ourworld.compuserve.com/homepages/pgubbins/tutmonda.htm
http://www2.netdoor.com/~lainh/SWtopten.html
http://www.fokus.gmd.de/linux/
http://www.synicon.com.au/sw/mf/falcon.htm
http://www.safelink.net/danrose/mp3.html
